modules/adapterremoval/main.nf

```diff
@@ -38,10 +37,25 @@ process ADAPTERREMOVAL {
             $adapterlist \\
             --basename ${prefix} \\
             --threads ${task.cpus} \\
-            --settings ${prefix}.log \\
+            --settings ${prefix}.settings \\
```

Comment:
```suggestion
```

Ok, I forgot that infact `$basename.settings` is default, so we can just remove it entirely.